artemis-hawtio/artemis-plugin/src/main/webapp/plugin/js/components/browse.js

```diff
@@ -326,7 +326,7 @@ var Artemis;
                 header: 'User ID'
             },
             {
-                itemField: 'StringProperties',
+                itemField: 'validatedUser',
                 header: 'Validated User',
                 templateFn: function(value) {
                     return value._AMQ_VALIDATED_USER;
```

Comment:
The line `header: 'Validated User'` was already in main. I don't know what it does -- I don't see it anywhere on the UI. I change the `itemField` name for consistency.  I'm happy to try and work on this. Let me know how you want it to work.